### DISCUSSION TITLE
第4节课笔记(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#285] [笔记]
16班第四节课笔记提交在下面

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17960310

**Comment by hhhanqian** [comment]
http://xhslink.com/SFgeiz

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-04/

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch04

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9200007

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/XyB8dq7bwoLp9axRRD8cXYxknqd?from=from_copylink

**Comment by Durian-1111** [comment]
**Day04-LearningNote**
链接：https://xby48gs45u.feishu.cn/docx/Q5bEdMuhcotHKdxYG8vcomrrn9c?from=from_copylink

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7323135762798952459

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/fmy4yr0mueq8rocm?singleDoc#

**Comment by Star-cre** [comment]
https://www.bilibili.com/video/BV1yK4y1B75J/?vd_source=2fb638751797274bd22bea982387a179  公开笔记

**Comment by Dhaizei** [comment]
http://t.csdnimg.cn/dzsWZ
